pkg/apis/serving/v1alpha2/framework_onnx.go

```diff
@@ -77,7 +77,7 @@ func (s *ONNXSpec) Validate(config *InferenceServicesConfig) error {
 	if err != nil {
 		return err
 	}
-	if ext := path.Ext(uri.Path); ext != ONNXFileExt || ext != "" {
+	if ext := path.Ext(uri.Path); ext != ONNXFileExt && ext != "" {
```

Comment:
Could you help add a unit test?

Reply:
yes, added here https://github.com/kubeflow/kfserving/pull/1140/commits/260e4f29c33afadfe5dd341fd2f3d95b2e43f0af, probably should have done that the first time :grimacing: